lib/associations/belongs-to-many.js

```diff
@@ -287,6 +287,16 @@ class BelongsToMany extends Association {
        */
       removeMultiple: 'remove' + plural,
       /**
+       * Un-associate all instances.
+       *
+       * @param {Object} [options] Options passed to `through.destroy`
```

Comment:
I just said the API should be similar, accepting the same parameters. @mickhansen @janmeier?